config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  get root "search#new"
+  get "recipes/", to: "recipes#index", as: "recipes_index"
+  get 'recipes/:uri/view', to: "recipes#show", as: "recipe_show"
+
+  get '/search', to: 'search#new', as: "search_recipes" #where searches are routed to
```

Comment:
I'm not sure that `new` is a good name for this route/action. You're not really creating a new recipe, you're doing a search.